### PR TITLE
New account management section

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adds a new account management section to Timescale Cloud, that covers things in the Billing details tab.
 
# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

- Fixes https://github.com/timescale/docs/issues/516
- Fixes https://github.com/timescale/docs/issues/451

### CODE REVIEW

File: cloud/account-management.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+## Add or change payment methods
+If you haven't yet added a payment method, you can add one in this section. You
+can also update to a new credit card here.
+
+<procedure>
+
+### Adding a payment method
+1.  [Log in to your Timescale Cloud account][cloud-login] and navigate to
+    the `Billing` details section.
+1.  Click `Add a payment method`. If you already have a payment method saved,
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply cloud/account-management.md:39]: Yeah, my original plan was to do it as two procedures, but when I actually did it, this is how it works. If I wrote it as two procedures, the only difference is in step two, where you remove the existing credit card first. Probably the title should be "Adding or changing a payment method" though.

File: cloud/account-management.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Account management
+You can manage billing for your Timescale Cloud account by navigating to the
+`Billing` details section in your Cloud console. From here, you can see your
+monthly usage, past payments you have made, the estimated cost of your next
+invoice, and your current payment method.
+
+## About Timescale Cloud billing
+Timescale Cloud invoices are issued on the first day of every month, and cover
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply cloud/account-management.md:8]: OK, I'll update this when I get access to the test system 👍

**Comment by Loquacity** [comment]
image
image

**Review by lukas-bernert — APPROVED** [review]

**Comment by Khyme** [comment]
> A confirmation email is sent to the email address you entered, as well as to the original email address that you signed up with.
Confirmation /validation (with a button to click) is sent only to the entered email address.
When clicked,
> Another email is sent to notify that the new email address is saved.
this notification only goes to signup address

File: cloud/account-management.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+When you add a billing email address, payment confirmations are sent to both the
+email address you add, and the original email address that you signed up with.
+</highlight>
+
+<procedure>
+
+### Adding a billing email address
+1.  [Log in to your Timescale Cloud account][cloud-login] and navigate to
+    the `Billing` details section.
+1.  In the `Payment email` tab, click `Add New Email`.
```
Comment (m0nae) [inline cloud/account-management.md:78]:
```suggestion
1.  In the `Payment emails` tab, click `Add New Email`.
```

File: cloud/account-management.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    signed up with.
+1.  Follow the instructions in the confirmation email to confirm the email
+    address. Another email is sent to notify that the new email address is
+    saved.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc-add-billing-email.png" alt="Adding a new billing email address in Timescale Cloud"/>
+
+</procedure>
+
+[cloud-login]: https://cloud.timescale.com
```
Comment (m0nae) [inline cloud/account-management.md:91]: Is https://cloud.timescale.com supposed to redirect to https://www.timescale.com/products? It does that for me.

**Review by m0nae — APPROVED** [review]
Added a couple comments but overall, looks good to me!

File: cloud/account-management.md (hunk 3 above)
Comment (Loquacity) [reply cloud/account-management.md:78]: It's definitely singular in the UI:

image

File: cloud/account-management.md (hunk 4 above)
Comment (Loquacity) [reply cloud/account-management.md:91]: Good spot! Updated to https://console.cloud.timescale.com/

File: cloud/account-management.md (hunk 3 above)
Comment (m0nae) [reply cloud/account-management.md:78]: It's not on dev just yet (PR is awaiting approval) but that was one of the changes I've made. I'll just ping you when the next version of the feature is on dev then?

Comment (Loquacity) [reply cloud/account-management.md:78]: Ah, OK! Didn't realise it had changed. I'll update the text now but I'll need to update the screenshot when it goes up.

**Review by jacobprall — APPROVED** [review]

**Comment by Loquacity** [comment]
lol @jacobprall I still need to update the screenshot 🤦‍♀️  I'll do a new PR ...

**Comment by jacobprall** [comment]
Sorry about that, totally my bad!!!

**Pull request merged**